### ISSUE TITLE
Back tooltip on personal information is in lowercase

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#17244]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. Go to the Sittings > Workspace > Connect bank account
2. Go to the Step 3 of 5 and Choose your document
3. After selecting a document type check "Back" tooltip

## Expected Result:
Back tooltip on personal information should be in uppercase

## Actual Result:
Back tooltip on personal information is in lowercase

## Workaround:
unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.2.98-2
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation
<img width="957" alt="Untitled" src="https://user-images.githubusercontent.com/43996225/231001887-ac89dad9-eba3-4111-b850-b9d25d7e393b.png">
![Screenshot_93](https://user-images.githubusercontent.com/43996225/231001908-4ea6afbc-d8df-4fb6-8009-23b2e6cdd6c1.png)
![Screenshot_91](https://user-images.githubusercontent.com/43996225/231001911-fa873ff1-3820-4a8b-87b1-4b2a00506eed.png)

https://user-images.githubusercontent.com/43996225/231001912-441937ef-3ed5-40fb-aa2c-37ed5a2a7a6a.mp4

**Expensify/Expensify Issue URL:**
**Issue reported by:** @ayazhussain79
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1681146014939409

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by MelvinBot** [comment]
Triggered auto assignment to @stephanieelliott (`Bug`), see https://stackoverflow.com/c/expensify/questions/14418 for more details.

**Comment by MelvinBot** [comment]
### Bug0 Triage Checklist ([Main S/O](https://stackoverflowteams.com/c/expensify/questions/14418))

- [ ] This "bug" occurs on a supported platform (ensure `Platforms` in OP are ✅)
- [ ] This bug is not a duplicate report (check [E/App issues](https://github.com/Expensify/App/issues) and [#expensify-bugs](https://expensify.slack.com/archives/C049HHMV9SM))
    - If it is, comment with a link to the original report, close the issue and add any novel details to the original issue instead
- [ ] This bug is reproducible using the reproduction steps in the OP. [S/O](https://stackoverflowteams.com/c/expensify/questions/14418#:~:text=The%20bug%20is%20reproducible)
    - If the reproduction steps are clear and you're unable to reproduce the bug, check with the reporter and QA first, then close the issue.
    - If the reproduction steps aren't clear and you determine the correct steps, please update the OP.
- [ ] This issue is filled out as thoroughly and clearly as possible
    - Pay special attention to the title, results, platforms where the bug occurs, and if the bug happens on staging/production.
- [ ] I have reviewed and subscribed to the linked Slack conversation to ensure Slack/Github stay in sync

**Comment by stephanieelliott** [comment]
Hey @kavimuru can you fill out the `Platforms` section of the issue body please? Once you do, I can test!

**Comment by Victor-Nyagudi** [comment]
It looks like the back button is from the `<Onfido />` element obtained from the Onfido Web SDK. 

This happens inside `RequestorOnfidoStep.js`. 

```javascript
//...
<ScrollView contentContainerStyle={styles.flex1}>
    <Onfido
        sdkToken={this.props.onfidoToken}
        onUserExit={() => {
            BankAccounts.clearOnfidoToken();
            BankAccounts.goToWithdrawalAccountSetupStep(CONST.BANK_ACCOUNT.STEP.REQUESTOR);
        }}
        onError={() => {
        // In case of any unexpected error we log it to the server, show a growl, and return the user back to the requestor step so they can       try again.
            Growl.error(this.props.translate('onfidoStep.genericError'), 10000);
            BankAccounts.clearOnfidoToken();
            BankAccounts.goToWithdrawalAccountSetupStep(CONST.BANK_ACCOUNT.STEP.REQUESTOR);
        }}
        onSuccess={(onfidoData) => {
            this.submit(onfidoData);
        }}
     />
</ScrollView>
//...
```  

The [Web SDK Flow](https://developers.onfido.com/guide/sdk-customization#web-sdk-flow) has a configuration option called `forceCrossDevice` that has been set to `true` which then shows the above-mentioned `<Onfido />` element prompting the user to continue on mobile after selecting a document.

Onfido offers some [UI customization options](https://developers.onfido.com/guide/sdk-customization#ui-customization) which have already been tweaked to match Expensify's design inside `BaseOnfidoWeb.js` - this is also where the config options are set - however, I don't see any option to change the back button's case from their docs.

The customization options for the document screen in question are limited to colors, fonts, and maybe border radii. 

The only `string` that appears to be customizable is that of the [optional welcome screen](https://developers.onfido.com/guide/sdk-customization#ui-customization) which I believe Expensify isn't currently using on Web.   

I'm not sure if this counts as a proposal per se, but I think changing the back button's case is an option Onfido hasn't provided. If it's accepted as one, then well and good - just thought I'd offer some insight from my findings. 

By the way, @kavimuru, did you mean title case instead of upper case for the back button?

- `back` - lowercase
- `Back` - title case
- `BACK` - uppercase

**Comment by MelvinBot** [comment]
📣 @Victor-Nyagudi! 📣

Hey, it seems we don’t have your contributor details yet! You'll only have to do this once, and this is how we'll hire you on Upwork.
Please follow these steps:

1. Get the email address used to login to your Expensify account. If you don't already have an Expensify account, create one [here](https://new.expensify.com/). If you have multiple accounts (e.g. one for testing), please use your main account email.
2. Get the link to your Upwork profile. It's necessary because we only pay via Upwork. You can access it by logging in, and then clicking on your name. It'll look like [this](https://www.upwork.com/freelancers/~01ca8ad8ce36b9eceb). If you don't already have an account, sign up for one [here](https://www.upwork.com/nx/signup/?dest=home).
3. Copy the format below and paste it in a comment on this issue. Replace the placeholder text with your actual details.

![Screen Shot 2022-11-16 at 4 42 54 PM](https://user-images.githubusercontent.com/30609178/202326034-1a292e64-48fd-4fef-9f4f-7feaaeedfd9f.png)

Format:
```
Contributor details
Your Expensify account email: <REPLACE EMAIL HERE>
Upwork Profile Link: <REPLACE LINK HERE>
```

**Comment by Victor-Nyagudi** [comment]
Contributor details
Your Expensify account email: vicktests4@gmail.com
Upwork Profile Link: https://www.upwork.com/freelancers/~0136b5e59ffbb32395

**Comment by MelvinBot** [comment]
✅ Contributor details stored successfully. Thank you for contributing to Expensify!

**Comment by kavimuru** [comment]
@stephanieelliott updated the platform section.

**Comment by MelvinBot** [comment]
@stephanieelliott Uh oh! This issue is overdue by 2 days. Don't forget to update your issues!

**Comment by stephanieelliott** [comment]
Hm, interesting proposal @Victor-Nyagudi! Adding the `External` label so we can get more eyes on it.

**Comment by MelvinBot** [comment]
Job added to Upwork: https://www.upwork.com/jobs/~014397c5d33abcb652

**Comment by MelvinBot** [comment]
Current assignee @stephanieelliott is eligible for the External assigner, not assigning anyone new.

**Comment by MelvinBot** [comment]
Triggered auto assignment to Contributor-plus team member for initial proposal review - @mollfpr (`External`)

**Comment by MelvinBot** [comment]
Triggered auto assignment to @neil-marcellini (`External`), see https://stackoverflow.com/c/expensify/questions/7972 for more details.

**Comment by mollfpr** [comment]
Reviewing now...

**Comment by mollfpr** [comment]
Thank you guys for the proposals!

As @Victor-Nyagudi and @aman-atg point out, that is a default from the Onfido SDK, and asking the Onfido maintainer to change the copy will not be a priority for them.

**Comment by mollfpr** [comment]
@Prince-Mendiratta The documentation links for iOS and Android you attach is pointing to the web SDK. Do you mean we need to do this https://github.com/onfido/react-native-sdk#android https://github.com/onfido/react-native-sdk#ios for the native to add custom localization?

**Comment by mollfpr** [comment]
@Prince-Mendiratta So, we will update the iOS and Android with the Onfido RN SDK, right? As the documentation is pretty clear about custom the localization on native platform, it will do for you to work on it?

**Comment by neil-marcellini** [comment]
I also like @Prince-Mendiratta's [proposal](https://github.com/Expensify/App/issues/17244#issuecomment-1509220752). Since the issue is only on Web and Desktop, we only need to implement the fix there. Let's keep it simple for now and only fix the capitalization of the back button. If you like, you could make a feature request for us to localize the entire Onfido flow, but I'm not sure it's a priority.

**Comment by MelvinBot** [comment]
📣 @Prince-Mendiratta You have been assigned to this job by @neil-marcellini!
Please apply to this job in Upwork and leave a comment on the Github issue letting us know when we can expect a PR to be ready for review 🧑‍💻
Keep in mind: [Code of Conduct](https://github.com/Expensify/App/blob/main/CODE_OF_CONDUCT.md) | [Contributing](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md) 📖

**Comment by stephanieelliott** [comment]
PR is on staging

**Comment by MelvinBot** [comment]
`Reviewing` label has been removed, please complete the "BugZero Checklist".

**Comment by MelvinBot** [comment]
The solution for this issue has been :rocket: deployed to production :rocket: in version 1.3.3-2 and is now subject to a 7-day regression period :calendar:. Here is the list of pull requests that resolve this issue:

- https://github.com/Expensify/App/pull/17602


If no regressions arise, payment will be issued on 2023-04-28. :confetti_ball:

After the hold period is over and BZ checklist items are completed, please complete any of the applicable payments for this issue, and check them off once done.
- [ ] External issue reporter
- [ ] Contributor that fixed the issue
- [ ] Contributor+ that helped on the issue and/or PR

As a reminder, here are the bonuses/penalties that should be applied for any External issue:
- Merged PR within 3 business days of assignment - 50% **bonus**
- Merged PR more than 9 business days after assignment - 50% **penalty**

**Comment by MelvinBot** [comment]
**BugZero Checklist:** The PR fixing this issue has been merged! The following checklist ([instructions](https://stackoverflow.com/c/expensify/questions/14574)) will need to be completed before the issue can be closed:

- [ ] [@mollfpr] The PR that introduced the bug has been identified. Link to the PR:
- [ ] [@mollfpr] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment:
- [ ] [@mollfpr] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion:
- [ ] [@stephanieelliott] Determine if we should create a regression test for this bug.
- [ ] [@mollfpr] If we decide to create a regression test for the bug, please propose the [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) steps to ensure the same bug will not reach production again.
- [ ] [@stephanieelliott] Link the GH issue for creating/updating the regression test once above steps have been agreed upon:

**Comment by sobitneupane** [comment]
[PR of this issue](https://github.com/Expensify/App/pull/17602) caused https://github.com/Expensify/App/issues/17789 regression.

**Comment by neil-marcellini** [comment]
@Prince-Mendiratta would you please submit a PR to fix the regression?

**Comment by stephanieelliott** [comment]
https://github.com/Expensify/App/pull/17943#issue-1682183369 is still undergoing QA, "waiting for payment" period will restart once it is merged.

**Comment by stephanieelliott** [comment]
PR on staging

**Comment by mollfpr** [comment]
> [@mollfpr] The PR that introduced the bug has been identified. Link to the PR:
 [@mollfpr] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment:
 [@mollfpr] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion:

Another improvement to Onfido languages, so no offending PR.

> [@mollfpr] Determine if we should create a regression test for this bug.

Maybe we can add the check to _**Workspace - Add bank account - Plaid back account selector screen**_?

**Comment by mollfpr** [comment]
Propose update for [Workspace - Add bank account - Plaid back account selector screen](https://expensify.testrail.io/index.php?/suites/view/7722&group_by=cases:section_id&group_order=asc&display_deleted_cases=0) TC added a step 14-18

1. Log in with a applause.expensifail account (that does not have any bank account already added)
2. Enable staging Web secure server in Account Preferences if disabled
3. Navigate to the add bank account modal (Workspace settings > Add bank account)
4. Select the "Connect with plaid" method to add a bank account
5. On the bank account list select "Wells Fargo". You'll be redirected to the "First Platypus Bank"
6. Enter the credentials "user_good / pass_good"
7. If a verification is prompted - select mobile and enter "credentials_good" as the verification mode
8. Checkmark the "Plaid Checking" and "Plaid Saving"
9. Checkmark the 2 options under "additional information you want to share" and submit
10. Check the T&C and click "Connect account information" and submit
11. Click on the last confirmation on the plaid modal
12. Verify you're redirected to the "choose an account" screen in NewDot
13. Select the bank account ending in 0000 but do not proceed to the next screen
14. Hover on the "back" arrow button
15. Verify that it's show the text "Back"
16. Change the account language preference to Spanish
17. Hover again on the "back" arrow button
18. Verify that it's show the text "Volver"
19. Click on the "back" arrow from the right panel and redo [4-12]
20. Verify the selector shows "choose an account" and there is no button below
21. Select the bank account ending in 1111
22. Verifya submit button appears below the selector
23. Click on the button
24. Verifyyou see the "Company Information" form

**Comment by stephanieelliott** [comment]
Hey @Prince-Mendiratta @mollfpr @ayazhussain79 can you please apply for the job in Upwork so that we can pay this out? Thank you!

**Comment by ayazhussain79** [comment]
@stephanieelliott  This job is no longer available [link](https://www.upwork.com/jobs/~014397c5d33abcb652), can you please send the invitation [here](https://www.upwork.com/freelancers/~018a62c1e70a3fab40)

**Comment by michaelhaxhiu** [comment]
Jumping in on behalf of Steph.

- Bug reporter @ayazhussain79 = $250
- Contributor @Prince-Mendiratta = $1000 (we had a regression, which is now fixed)
- C+ @mollfpr = $1000 

New job post - https://www.upwork.com/jobs/~012bae1e57414f2d48

**Comment by MrJithil** [comment]
Are you still looking for someone to fix this? If yes, I can land a PR in 24 hours.

**Comment by ayazhussain79** [comment]
@michaelhaxhiu Offer accepted

**Comment by MrJithil** [comment]
Contributor details
Your Expensify account email: jithil@outlook.com
Upwork Profile Link: https://www.upwork.com/freelancers/~01efe261118bd962c0

**Comment by MrJithil** [comment]
Im on it.

**Comment by michaelhaxhiu** [comment]
No sorry. This job is already done and we are paying the contributors who solved it.

@MrJithil if you are interested in taking on a job, please filter by `Help Wanted` in the E/App repo:

https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22

**Comment by michaelhaxhiu** [comment]
@MrJithil No problem! I just noticed you didn't request an invite to our slack channels yet. I recommend reading this page real quick to get started with contributing:

https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#propose-a-solution-for-the-job
